website/src/templates/markdown.js

```diff
@@ -322,15 +323,17 @@ const components = {
 
 const MarkdownTemplate = props => (
   <LayoutContent pageData={props.data.mdx}>
-    <MDXProvider components={components}>
-      <TypographyPage>
-        <SEO title={props.data.mdx.frontmatter.title} />
-        {/* This wrapper div is important to ensure the vertical space */}
-        <div>
-          <MDXRenderer>{props.data.mdx.code.body}</MDXRenderer>
-        </div>
-      </TypographyPage>
-    </MDXProvider>
+    <IntlProvider locale="en">
```

Comment:
I don't think we need this, it's defined now in the `example-wrapper.js`

Reply:
Yeah, forgot to remove this after the latest rebase